docs/conf.py

```diff
@@ -55,4 +56,4 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#  html_static_path = ['_static']
```

Comment:
If I understand correctly, this was commented out because it was not being used, right?

Reply:
yes and also remove the warning that `_static` folder is not exist